jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/FileBufferedResponseHandler.java

```diff
@@ -193,41 +204,27 @@ private void commit(Callback callback)
             }
 
             // Create an iterating callback to do the writing
-            IteratingCallback icb = new IteratingCallback()
-            {
-                private final long fileLength = _filePath.toFile().length();
-                private long _pos = 0;
-                private boolean _last = false;
+            ByteBufferPool.Sized sizedPool = ByteBufferPool.Sized.as(getServer().getByteBufferPool(), true, getBufferSize());
+            Content.Source source = Content.Source.from(sizedPool, _filePath);
```

Comment:
This does not `mmap` files naymore, but honestly I think it is for the best.

Reply:
Supporting mmap from Content.Source.from(path) is still possible, but in a future PR.